src/wiki/getting-started/install-of-alternatives.md

```diff
@@ -33,12 +33,11 @@ If you would like to go a bit further, LamdAurora maintains a very detailed list
 
 # Fabulously Optimized
 
-If you don't want to search, and install these mods manually, then the [Fabulously Optimized](https://github.com/Fabulously-Optimized/fabulously-optimized) modpack is for you!
+If you don't want to search, and install these mods manually, then give a try to the [Fabulously Optimized](https://github.com/Fabulously-Optimized/fabulously-optimized) modpack.
```

Comment:
`give a try to` doesn't sound right :eyes:

Reply:
how should it be then lol

Reply:
Just `try` :sweat_smile: 
`...then try the...`

Reply:
done